client/lib/wp/sync-handler/README.md

```diff
@@ -19,3 +19,25 @@ const handler = new SyncHandler( wpcomXHRHandler );
 // create wpcom instance passing the wrapped handler
 const wpcom = wpcomUndocumented( handler );
 ```
+
+### Cache Invalidation
+
+`cache` attempts cleanup the obsolete records from the local storage of the browser so thus reduce the dB size. The current criteria are two:
+
+### syncHandler#pruneRecordsFrom( [lifetime] );
+Prune records older than the given `lifetime` (milliseconds or natural language). By default the value of the lifetime is `2 days`.
```

Comment:
If we're going to use the `ms` library, we should link to the docs for `ms` here (just turn the words "natural language" into a link) so it's easy for people to see what values are allowed.